bindgen/lib.rs

```diff
@@ -87,10 +87,10 @@ pub const DEFAULT_ANON_FIELDS_PREFIX: &str = "__bindgen_anon_";
 const DEFAULT_NON_EXTERN_FNS_SUFFIX: &str = "__extern";
 
 fn file_is_cpp(name_file: &str) -> bool {
-    name_file.ends_with(".hpp") ||
-        name_file.ends_with(".hxx") ||
-        name_file.ends_with(".hh") ||
-        name_file.ends_with(".h++")
+    name_file.ends_with(".hpp")
```

Comment:
Oh... This must be my rustfmt settings. Strange, I'd have expected my IDE to grab them from the rustfmt.toml at the top

Reply:
you need nightly rust to format bindgen's code